frontend/lib/src/components/elements/Json/Json.tsx

```diff
@@ -57,7 +63,7 @@ export default function Json({ width, element }: JsonProps): ReactElement {
 
   // Try to pick a reasonable ReactJson theme based on whether the streamlit
   // theme's background is light or dark.
-  const jsonTheme = hasLightBackgroundColor(theme) ? "rjv-default" : "monokai"
+  const jsonTheme = getJsonTheme(theme)
```

Comment:
Nit: Think this was fine/cleaner the way it was?

Reply:
I put it into a separate method so that I could test just that single method. However, I will revert these changes as it looks like we want to test the actual theme.